docs/DeveloperGuide.md

```diff
@@ -168,6 +168,36 @@ according to an option and shows the updated list.
 The following diagram illustrates how the sort function operates:
 ![SortSequenceDiagram](images/SortSequenceDiagram.png)
 
+### \[Implemented\] Statistics feature
+
+The statistics feature is supported by `LogicManager` and `ModelManager`.
+
+To show flashcard statistics, `LogicManager` first calls `FlashBackParser#parseCommand` to parse through user input.
+If user input is recognized as a command to display statistics, `StatsCommandParser#parse` is invoked to create
+a new `StatsCommand` object.
+
+The `StatsCommand` is then executed:
+* The current flashcard list is obtained from the `ModelManager`.
+
+
+* If a valid flashcard index was included in the user input, the flashcard
+identified by the provided index is retrieved from the current flashcard list, and the statistics associated with the card is obtained by
+`Flashcard#getStats()`.
+
+* If the flashcard index was omitted from the user input. A new `Statistics` object is created, representing the
+overall statistics of the current flashcard list.
+
```

Comment:
You can add which methods you implement for this stats command, you can refer to sort and undo/redo implementation for more details "It implements the following operations:" part.

Reply:
Great suggestion. I have updated the section accordingly.